docker-compose.yml

```diff
@@ -244,6 +244,7 @@ services:
     environment:
       ASSETS_DIRECTORY_OVERRIDE: /etc/georchestra/datahub/assets
       CONFIG_DIRECTORY_OVERRIDE: /etc/georchestra/datahub/conf
+      CUSTOM_SCRIPTS_DIRECTORY: /etc/georchestra/datahub/scripts
```

Comment:
Is there custom scripts for datahub included by default in the datadir? Because if so, it doesn't make sense to include this variable by default.
And it would make more sense to comment it.

Reply:
I add the same reflexion but in fact we want the header of datahub working by default so we need this

Reply:
https://github.com/georchestra/datadir/tree/docker-master/datahub/scripts
here the script are in the datadir by default

Reply:
Ok correct